docker-compose.dev.yml

```diff
@@ -2,8 +2,13 @@ version: '3.4'
 
 services:
   identifier:
-    ports:
-      - "80:80"
```

Comment:
I still need those ports but I can put them in my override if it's annoying for you !

Reply:
Oh yeah my bad I always forget to add it back 😅. Maybe better indeed to add it in the override.